app/Http/Controllers/FrontController.php

```diff
@@ -164,8 +164,9 @@ public function partners()
      */
     public function campaigns(Request $request)
     {
-        $featured = Campaign::public()->front()->featured()->get();
+        $featured = Campaign::public()->where('is_hidden', 0)->front()->featured()->get();
```

Comment:
It would be even better to just add the condition to the `frontScope` on campaigns, so we don't have to remember to add it in multiple places ;-)